tests/integration-test.sh

```diff
@@ -32,23 +32,33 @@ else
     export FAST_FAILURE=0
 fi
 
+if (echo $prBody | grep -q "\[shared datastore\]"); then
+    export TEST_DATASTORE='cloud_native_1tb_01'
```

Comment:
is this the right place for the actual datastore name? It looks like we're using variables throughout the rest of the script, so:
```bash
export TEST_DATASTORE=${SHARED_DATASTORE}
```
?